integration_tests/testsuite/test_util.py

```diff
@@ -97,7 +115,7 @@ def _post(url, payload, timeout=DEFAULT_TIMEOUT):
     except requests.exceptions.Timeout:
         logging.error('POST to {0} timed out after {1} seconds!'
                       .format(url, timeout))
-        return 1
+        return 'ERROR', 1
 
 
 def _check_response(response, error_message):
```

Comment:
Any particular reason for making this change? Multiple return values here feels a little easy to mess up.